engine/render/wscript

```diff
@@ -30,15 +31,16 @@ def configure(conf):
 
     conf.recurse('src')
 
+    target_os = conf.env['TARGET_OS']
     platform = conf.env['PLATFORM']
 
-    if platform in "linux":
+    # TODO: check if here was mistake - if platform in "linux":
+    if target_os == TargetOS.LINUX:
         conf.env.append_value('CXXFLAGS', '-DGL_GLEXT_PROTOTYPES')
 
-    if "win32" in platform:
+    if target_os == TargetOS.WINDOWS:
         conf.env.append_value('LINKFLAGS', ['opengl32.lib', 'user32.lib', 'shell32.lib'])
```

Comment:
I'm not sure this is needed, as it's been working well for x86_64 without these flags?
Try removing the code?

I believe all the necessary link flags are setup with the `SOCKET APP` in render/src/test/wscript